Mathlib/Data/Rel.lean

```diff
@@ -262,8 +355,42 @@ namespace Function
 def graph (f : α → β) : Rel α β := fun x y => f x = y
 #align function.graph Function.graph
 
+theorem graph_id : graph id = @Eq α := by simp[graph]
```

Comment:
Since we're unfolding the definition of `graph` quite a lot, let's add a lemma to do that for us:
```lean
@[simp] lemma graph_def (f : α → β) (x y) : f.graph x y ↔ (f x = y) := Iff.rfl
```